src/tools/aml-language-server/util.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import * as url from 'url';
+
+export const jsonrpc = '2.0';
+
+export interface AmlServiceOptions {
+  log: string;       // The logging service to use (either 'console' or 'null').
+  port: number;      // The port number to use for tcp/pi communication.
+  help: boolean;     // Print the help information and do not run the server.
+  version: boolean;  // Print the version information and do not run the server.
+  stdio: boolean;    // Use stdio (rather than a tcp/ip port).
+}
+
+export interface AmlServiceContext {
+  logger: Logger;
+  rootPath?: string;
+  options: AmlServiceOptions;
+}
+
+export interface Logger {
+  // tslint:disable-next-line: no-any
```

Comment:
Rather than having this on every line, just put `tslint:disable: no-any` above Logger and `tslint:enable: no-any` below DevNullLogger.

Reply:
Done